types/three/src/animation/KeyframeTrack.d.ts

```diff
@@ -8,10 +8,10 @@ export class KeyframeTrack {
     /**
      * @param name
      * @param times
-     * @param values
+     * @param [values]
      * @param [interpolation=THREE.InterpolateLinear]
      */
-    constructor(name: string, times: ArrayLike<number>, values: ArrayLike<any>, interpolation?: InterpolationModes);
+    constructor(name: string, times: ArrayLike<number>, values?: ArrayLike<any>, interpolation?: InterpolationModes);
```

Comment:
Does this mean that the `values` property should now include `| undefined`?

Reply:
No, the `KeyframeTrack.values` does not require to be a Float32Array, any arraylike object is available.
In source code, it use [AnimationUtils.convertArray](https://github.com/mrdoob/three.js/blob/master/src/animation/KeyframeTrack.js#L21) to transform an array to be a Float32Array (by default) and any other types array according to `KeyframeTrack.prototype.ValueBufferType` (its default value is a Float32Array).

Reply:
Right I'm asking about [the property on the class](https://github.com/three-types/three-ts-types/blob/master/types/three/src/animation/KeyframeTrack.d.ts#L18) after it's been set by the constructor.